ungrib/src/ngl/g2/intmath.f

```diff
@@ -169,7 +169,7 @@ function ilog2_2(i_in)
       ilog2_2=0
       i=i_in
       if(i<=0) return
-      if(iand(i,i-1)/=0) then
+      if(iand(i,i-1_2)/=0) then
          !write(0,*) 'iand i-1'
          ilog2_2=1
```

Comment:
Do we need to change this assignment from
```
ilog2_2=1
```
to 
```
ilog2_2=1_2
```
All of the other integers are declared that way.

Reply:
There should be no problem here to let the compiler perform the type conversion.